src/pages/miyeon/Login/Login.js

```diff
@@ -1,9 +1,105 @@
 import React from 'react';
+import { Link, withRouter } from 'react-router-dom';
+import './Login.scss';
 
 class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      idVal: '',
+      pwVal: '',
```

Comment:
```suggestion
      idValue: '',
      pwValue: '',
```
애매하게 줄이기보다는 차라리 정확하게 적어주는게 좋을거 같아요